pandas/io/excel/_base.py

```diff
@@ -346,33 +353,33 @@
 @Appender(_read_excel_doc)
 def read_excel(
     io,
-    sheet_name=0,
-    header=0,
+    sheet_name: str | int | list[int] | list[str] | None = 0,
+    header: int | Sequence[int] | None = 0,
     names=None,
-    index_col=None,
+    index_col: int | Sequence[int] | None = None,
     usecols=None,
-    squeeze=None,
+    squeeze: bool | None = None,
     dtype: DtypeArg | None = None,
-    engine=None,
+    engine: Literal["xlrd", "openpyxl", "odf", "pyxlsb"] | None = None,
     converters=None,
-    true_values=None,
-    false_values=None,
-    skiprows=None,
-    nrows=None,
+    true_values: Iterable[Hashable] | None = None,
+    false_values: Iterable[Hashable] | None = None,
```

Comment:
The documentation says `list` but any iterable might work as well :)

Reply:
Yep, this is passed into ``maybe_convert_bool`` which accepts an Iterable